lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -1103,10 +1105,14 @@ def _get_node_ip_address(
         while True:
             addr = self._try_get_node_ip_address(environment, log, node)
             if addr:
+                log.debug(f"Domain {node_context.vm_name} booted with IP - {addr}")
                 return addr
 
             if time.time() > timeout:
-                raise LisaException(f"no IP addresses found for {node_context.vm_name}")
+                raise LisaException(
+                    f"no IP addresses found for {node_context.vm_name}."
+                    " Domain creation might have failed"
```

Comment:
What do you mean by "domain creation"?

Typically, when I see this error, it is because the OS failed to boot.

Reply:
`Domain` is Libvirt's term to mean a VM/Guest. Since we are using Libvirt internally for VM creation, I added the message this way.

Reply:
As you suggested in other comment, "Libvirt domain" would be apt to use. Will change it.